### PR TITLE
Update expected data as double precision data

### PR DESCRIPTION
**Pull request by BruceDai**
I tested on Intel TGL-i7 1165G7 laptop / AMD Ryzen 4800U/5800U laptops / Intel i9-9900K desktop etc., all tests passed with updated expected data and updated `utils.checkValue` function.

Next I plan to update WPT op tests based on these updated expected data to collect ULP data.

@huningxin PTAL, thanks.

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -2,43 +2,10 @@
...
-  }
-}
-
-export function checkValue(
-    tensor, expected, criteria = opFp32AccuracyCriteria) {
+export function checkValue(tensor, expected) {
   assert.isTrue(tensor.size === expected.length);
   for (let i = 0; i < expected.length; ++i) {
-    assert.isTrue(almostEqual(tensor.getValueByIndex(i), expected[i], criteria));
+    assert.equal(tensor.getValueByIndex(i), expected[i]);
```
Comment (huningxin) [inline test/utils.js]: I suppose we should use `almostEqual` for float values comparison even the calculation is in double precision.

Could you use the ULP-based comparison and set the ULP tolerance to 0?

Comment (BruceDai) [reply test/utils.js]: @huningxin I think that the expected data here is indeed the finally output data which we could directly use it by WPT tests, that is,  we will directly convert these tests of this webnn-baseline repo to WPT. WDYT? Thanks.

Comment (huningxin) [reply test/utils.js]: You could use these double precision expected data for WPT test as the baseline results. However, here these data are used to test the baseline implementation. You can image that these code may produce slightly different value on different JavaScript engine or architecture. So as I mentioned, if it compares float points, we should use `almostEqual` which could be ULP based.

Comment (BruceDai) [reply test/utils.js]: OK, I will update it by ULP based comparison. Thanks.

Comment (BruceDai) [reply test/utils.js]: @huningxin I've updated the commit, please take another look, thanks.

File: test/utils.js [hunk 2]
```diff
@@ -2,43 +2,42 @@
...
+  return distance > nulp;
 }
 
-export function checkValue(
-    tensor, expected, criteria = opFp32AccuracyCriteria) {
+export function checkValue(tensor, expected, nulp = 0n) {
   assert.isTrue(tensor.size === expected.length);
   for (let i = 0; i < expected.length; ++i) {
-    assert.isTrue(almostEqual(tensor.getValueByIndex(i), expected[i], criteria));
+    assert.isFalse(compareUlp(tensor.getValueByIndex(i), expected[i], nulp));
```
Comment (huningxin) [inline test/utils.js]: `compareUlp` returning false is not intuitive here. Could you please rename the comparison method to `almostEqualUlp` and return true if they are close enough?

Comment (BruceDai) [reply test/utils.js]: Updated. PTAL, thanks.

**Review by huningxin — APPROVED** [review]
LGTM, thanks @BruceDai !

**Pull request merged**